.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = F401,F405,W503,F841,F403,E402,F821,W291,E266,F811
```

Comment:
Why so much error code in ignoring. If there are errors then they should be fixed in this PR.
You can consider trailing commas and import in `__init__.py` error code in ignoring or any other code which is required but do mention about code.

Reply:
@codesankalp  should I mention error codes `.flake8` itself or in `README`?

Reply:
No just add how to run check for flake8.